### PR TITLE
Modify help message

### PR DESCRIPTION
**Pull request by sinhaashish**

### CODE REVIEW

File: cmd/kubectl-directpv/kubectl-direct_csi.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func main() {
 		os.Exit(1)
 	}()
 
-	if filepath.Base(os.Args[0]) == "kubectl-direct_csi" {
+	if filepath.Base(os.Args[0]) == "kubectl-directpv" {
```
Comment (wlan0) [inline cmd/kubectl-directpv/kubectl-direct_csi.go:46]: this checks for old binary name and shows the deprecation notice. The check should be for direct_csi, not directpv.

Comment (harshavardhana) [inline cmd/kubectl-directpv/kubectl-direct_csi.go:46]:
```suggestion
	if filepath.Base(os.Args[0]) == "kubectl-direct_csi" {
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/kubectl-directpv/drives.go [hunk 2]
```diff
@@ -57,6 +57,7 @@ func validateDriveSelectors() (err error) {
 }
 
 func init() {
+	// drivesCmd.SetUsageTemplate("kubectl ashish")
```
Comment (wlan0) [inline cmd/kubectl-directpv/drives.go:60]: delete

**Comment by sinhaashish** [comment]
Not relevant anymore

**Pull request closed without merging**